package.json

```diff
@@ -158,23 +158,28 @@
     "pretest": "npm run compile-tests && npm run compile && npm run lint",
     "check-types": "tsc --noEmit",
     "lint": "eslint src",
-    "test": "vscode-test"
+    "test": "mocha --import=tsx tests/**/*.test.ts --exit && vscode-test"
   },
   "devDependencies": {
+    "@types/chai": "^5.0.1",
     "@types/js-yaml": "^4.0.9",
```

Comment:
https://github.com/Mastersam07/maestro-workbench/pull/35/files#r1913424418

From the above, taking out `js-yaml` should take out `@types/js-yaml` no?

Reply:
Fixed